src/universal/modules/userDashboard/components/UserActionList/ActionDragLayer.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component, PropTypes} from 'react';
+import UserActionListItem from './UserActionListItem';
+import {DragLayer as dragLayer} from 'react-dnd';
+import ui from 'universal/styles/ui';
+
+const layerStyles = {
+  position: 'fixed',
+  pointerEvents: 'none',
+  zIndex: 500,
+  left: 0,
+  top: 0,
+  width: ui.dashActionsWidth,
+  minHeight: '15rem',
```

Comment:
So I think transform is probably the performant way to go. If we go with the tilt effect, I would probably knock it down a degree or two. I’m curious why there seems to be a border-radius on the bottom two corners.

I think an alternative to rotating would be to just scale it up to 1.1 or 1.2, and see what we think about that. Going to try a few things locally.